src/components/snapshot/overview/SnapshotExport.tsx

```diff
@@ -1,54 +1,30 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import { ClassNameMap, createStyles, withStyles } from '@mui/styles';
 import {
   Button,
+  Box,
   Checkbox,
   CircularProgress,
   FormControlLabel,
   FormGroup,
   FormHelperText,
   Typography,
 } from '@mui/material';
-import { CustomTheme } from '@mui/material/styles';
 import { exportSnapshot, resetSnapshotExport } from '../../../actions';
 import { TdrState } from '../../../reducers';
 import { AppDispatch } from '../../../store';
 import { CloudPlatform, SnapshotExportResponseModel, SnapshotModel } from '../../../generated/tdr';
 import { SnapshotRoles } from '../../../constants';
 
-const styles = (theme: CustomTheme) =>
-  createStyles({
-    card: {
-      display: 'inline-block',
-      padding: theme.spacing(4),
-      width: '100%',
-    },
-    exportButton: {
-      marginTop: '0.5rem',
-      height: '36px',
-    },
-    centered: {
-      textAlign: 'center',
-    },
-    labelRight: {
-      paddingLeft: '10px',
-    },
-    section: {
-      paddingBottom: theme.spacing(1),
-    },
-  });
-
-type SnapshotExportProps = {
-  classes: ClassNameMap;
-  dispatch: AppDispatch;
-  exportResponse: SnapshotExportResponseModel;
-  isDone: boolean;
-  isProcessing: boolean;
-  of: SnapshotModel;
-  terraUrl: string | undefined;
-  userRoles: Array<string>;
-};
+interface SnapshotExportProps {
+  readonly dispatch: AppDispatch;
+  readonly exportResponse: SnapshotExportResponseModel;
+  readonly isDone: boolean;
+  readonly isProcessing: boolean;
+  readonly of: SnapshotModel;
+  readonly terraUrl: string | undefined;
+  readonly userRoles: Array<string>;
+}
```

Comment:
Why was this change needed to be a readonly interface?

Reply:
IIRC, Sonarcloud complained about them not being `readonly`. I can't seem to find that report now - it doesn't look like the reports on older commits are retained.